server/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "plaid-node-walkthrough",
+  "version": "0.1.0",
+  "description": "A sample app and accompanying walkthrough for the Plaid API.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "developers@plaid.com",
```

Comment:
do we need this?

Reply:
Removed

Reply:
Package.json is needed to declare dependencies for the server